src/controller/SetUpCodePairer.h

```diff
@@ -93,9 +93,8 @@ class DLL_EXPORT SetUpCodePairer : public DevicePairingDelegate
     void SetBleLayer(Ble::BleLayer * bleLayer) { mBleLayer = bleLayer; };
 #endif // CONFIG_NETWORK_LAYER_BLE
 
-    // Called to notify us that the DeviceCommissioner is shutting down and we
-    // should not try to do any more new work.
-    void CommissionerShuttingDown();
+    // Stop ongoing discovery / pairing (of the specified node)
+    bool StopPairing(NodeId remoteId = kUndefinedNodeId);
```

Comment:
This might be clearer if we just use `Optional<NodeId> remoteId = NullOptional`.  It's a little less efficient, but shouldn't rely on placeholder values, etc....  Either way, though.

Reply:
Going to leave this as is since the vendor and product IDs are handled in the same way.